### PR TITLE
Feat/export create har

### PR DESCRIPTION
**Pull request by HenriPro**
This branch has already been used in production for almost 2 years.  (we used a `git://...`  in the package.json) see : https://github.com/Kong/swagger-ui-kong-theme/blob/master/package.json#L37

In order to make updates easier, lets bring the change to master. To do so I have also bumped the npm major version as this is a breaking change.  Once this PR is approved I will publish on npm.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "swagger2har",
-  "version": "0.1.2",
+  "version": "0.2.0",
```
Comment (chrisvdp) [inline package.json]: This is a minor bump

Comment (HenriPro) [reply package.json]: Oh whoops you are right, I think it's best to go to 1.0

**Review by chrisvdp — DISMISSED** [review]
LGTM

**Comment by mmorel-35** [comment]
Don't you forget to update the Readme?

**Comment by HenriPro** [comment]
> Don't you forget to update the Readme?

Good catch!

**Review by chrisvdp — APPROVED** [review]
Still LGTM

**Pull request merged**